### ISSUE TITLE
Survey prompt: move permalinks to stretch goals

### ISSUE THREAD

**Issue by rhjones** [ga-wdi-boston/group-project#165]
Talked to Jeff. He asked us to file an issue to move permalinks to the stretch goals for the survey prompt.

**Comment by jrhorn424** [comment]
This is correct. I believe permalinks should be a stretch requirement since the natural implementation of them would require using browser APIs we haven't discussed during the course.

**Comment by payne-chris-r** [comment]
@jrhorn424 can you embellish on this? I'm not sure what you mean by permalinks? How do you do it without permalinks?

**Comment by payne-chris-r** [comment]
This needs to be a stretch goal. I don't see how it's possible without FE routing. It could persuade us to move FE routing to before `team-project` though.

**Comment by jrhorn424** [comment]
This is starting to drift off-topic.

@gaand and I have discussed including client routing in the template by default.
